src/css/styles.scss

```diff
@@ -320,7 +330,7 @@ h3 {
 }
 
 main {
-  background-color: lighten($secondary-color, 10%);
+  background-color: $quinary-color;
```

Comment:
I think this color will make more sense to use since we had a hard time seeing the instructions.
Ther is a lot of clean up needed here